doc/src/advanced.rst

```diff
@@ -48,10 +48,6 @@ when maintaining long-lived TCP connections. Users are advised to renew the
 IDLE command every 10 minutes to avoid the connection from being abruptly
 closed.
 
-Handling large mailboxes
```

Comment:
Yes, this is not urgent, as long as an issue remains open for later.

Reply:
That's what I was thinking.